File: hw4/Facts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

class Facts {
	static long factorial(int x){
		int y;
		int result=1;
		for (y=1;y<(x+1);y++){
			result=result*y;
		}
return result;
	}
  public static void main(String args[]) {
    System.out.println("n\tn!");
    System.out.println("----------------");
    int n;
    for (n=0;  n<=5;  n++)
      System.out.println(n+"\t"+Facts.factorial(n));
    return;

  }
}

File: hw4/facts.py
# print square roots in Python language.  R. Brown, 9/2010 

from math import sqrt


def factorial(x):
	result=1
	for i in range(1,x+1):
		result=result*i
	return result

print("n\tn!\n----------------")

for n in range(0, 6):
    print(n,"\t",factorial(n))
